subsys/net/ip/net_tc_mapping.h

```diff
@@ -0,0 +1,138 @@
+/** @file
+ * @brief Various priority to traffic class mappings
+ *
+ * This is not to be included by the application.
+ */
+
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __TC_MAPPING_H
+#define __TC_MAPPING_H
+
+#include <zephyr/types.h>
+
+/* All the maps below use priorities and indexes, below is the list of them
+ * according to 802.1Q - table I-2.
+ *
+ *  Priority         Acronym   Traffic types
+ *  1 (lowest)       BK        Background
+ *  0 (default)      BE        Best effort
+ *  2                EE        Excellent effort
+ *  3                CA        Critical applications
+ *  4                VI        Video, < 100 ms latency and jitter
+ *  5                VO        Voice, < 10 ms latency and jitter
+ *  6                IC        Internetwork control
+ *  7 (highest)      NC        Network control
+ */
+
+/* Helper macros used to generate the map to use */
+#define PRIORITY2TC_GEN_INNER(TYPE, COUNT) priority2tc_ ## TYPE ## _ ## COUNT
+#define PRIORITY2TC_GEN(TYPE, COUNT) PRIORITY2TC_GEN_INNER(TYPE, COUNT)
+
+#if defined(CONFIG_NET_TC_MAPPING_STRICT)
+
+/* This is the recommended priority to traffic class mapping for
+ * implementations that do not support the credit-based sharper transmission
```

Comment:
Also "sharper". Probably worth doing search-and-replace...

Reply:
@pfalcon Right! Sorry about that! This is actually a copy of the same text (Header and Kconfig), thank you for letting me know :+1: I'll submit a patch for that.